src/frontend/efiling-frontend/src/modules/helpers/RoutingUtil.js

```diff
@@ -0,0 +1,4 @@
+export function redirectToParentApp() {
+  const parentAppUrl = sessionStorage.getItem("cancelUrl");
```

Comment:
the cancel url won't exist in that context, the parent app should provide a link, maybe in the url of view my submission ?
The cancel url is part of the other flow when they submit a package from a parent app.

Reply:
Okay, yeah, I was wondering about that. 

So you're suggesting the /filingpackages/{packageIdentifier} endpoint could also include a return URL in the links section?

Reply:
yeah let's discuss that, I think it should be part of the url

Reply:
?returnUrl=

Reply:
Done. I changed this so that it's now based on a **returnUrl** param instead. If it exists, the **Return to Parent App** button shows.

ie.
http://localhost:3000/efilinghub/packagereview/1?returnUrl=http://www.google.com